[0m15:09:52.124825 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f004fea2980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f004f4f16f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f004f448580>]}


============================== 15:09:52.135387 | 88f741a8-3297-402e-9be9-3bcfd1b0d2dd ==============================
[0m15:09:52.135387 [info ] [MainThread]: Running with dbt=1.6.1
[0m15:09:52.136107 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/home/vscode/.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m15:09:52.136853 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/vscode/.dbt')}
[0m15:09:52.137428 [info ] [MainThread]: Creating dbt configuration folder at 
[0m15:10:30.229884 [debug] [MainThread]: Starter project path: /home/vscode/.local/lib/python3.10/site-packages/dbt/include/starter_project
[0m15:10:30.233021 [info ] [MainThread]: 
Your new dbt project "coffee_shop_jeroenmeij" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:10:30.234112 [info ] [MainThread]: Setting up your profile.
[0m15:11:56.023643 [info ] [MainThread]: Profile coffee_shop_jeroenmeij written to /home/vscode/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m15:11:56.024789 [debug] [MainThread]: Command `dbt init` succeeded at 15:11:56.024638 after 123.93 seconds
[0m15:11:56.025213 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f004fea2980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f004f448580>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f004f4488b0>]}
[0m15:11:56.025879 [debug] [MainThread]: Flushing usage events
[0m15:43:35.092219 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f721542a9b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f72144dfd30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f72144dfcd0>]}


============================== 15:43:35.094783 | e52fb5e1-6a78-4a32-8672-9abcb3266e97 ==============================
[0m15:43:35.094783 [info ] [MainThread]: Running with dbt=1.6.1
[0m15:43:35.095525 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/home/vscode/.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m15:43:35.096049 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-jeroen/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:43:35.096865 [debug] [MainThread]: Command `dbt run` failed at 15:43:35.096671 after 0.03 seconds
[0m15:43:35.097358 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f721542a9b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f72144dfd30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f72144dff40>]}
[0m15:43:35.097774 [debug] [MainThread]: Flushing usage events
[0m15:46:19.914016 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e0f1e6980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e0e297f10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e0e297cd0>]}


============================== 15:46:19.916530 | 2d7865d8-3b9f-4443-bcc7-b7e85ccc31c4 ==============================
[0m15:46:19.916530 [info ] [MainThread]: Running with dbt=1.6.1
[0m15:46:19.917177 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/home/vscode/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt run', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:46:19.917877 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-jeroen/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:46:19.918619 [debug] [MainThread]: Command `dbt run` failed at 15:46:19.918511 after 0.03 seconds
[0m15:46:19.919010 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e0f1e6980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e0e297be0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e0e297d90>]}
[0m15:46:19.919441 [debug] [MainThread]: Flushing usage events
[0m09:07:08.741254 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fbcb316dd20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fbcb221fd00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fbcb221fb50>]}


============================== 09:07:08.750652 | 37e46a00-73ff-41f1-8ff3-595d53f4b633 ==============================
[0m09:07:08.750652 [info ] [MainThread]: Running with dbt=1.6.1
[0m09:07:08.751419 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/home/vscode/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt deps', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:07:08.752180 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-jeroen/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:07:08.752998 [debug] [MainThread]: Command `dbt deps` failed at 09:07:08.752854 after 0.04 seconds
[0m09:07:08.753673 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fbcb316dd20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fbcb2207ac0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fbcb22063b0>]}
[0m09:07:08.754083 [debug] [MainThread]: Flushing usage events
[0m15:41:32.593616 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9cc8bba9b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9cc7c43cd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9cc7c43ee0>]}


============================== 15:41:32.613459 | 9b469818-9231-4113-9d8d-3900d8fb07c4 ==============================
[0m15:41:32.613459 [info ] [MainThread]: Running with dbt=1.6.1
[0m15:41:32.614174 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/home/vscode/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:41:32.614782 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-jeroen/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:41:32.615619 [debug] [MainThread]: Command `dbt run` failed at 15:41:32.615450 after 0.06 seconds
[0m15:41:32.616193 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9cc8bba9b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9cc7c3c550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9cc7c3c6d0>]}
[0m15:41:32.616628 [debug] [MainThread]: Flushing usage events
